cmd/webhook/app/options/options.go

```diff
@@ -28,14 +28,22 @@ import (
 )
 
 // WebhookFlags defines options that can only be configured via flags.
-type WebhookFlags struct{}
+type WebhookFlags struct {
+	// Path to a file containing a WebhookConfiguration resource
+	Config string
+}
 
 func NewWebhookFlags() *WebhookFlags {
 	return &WebhookFlags{}
 }
 
 func (f *WebhookFlags) AddFlags(fs *pflag.FlagSet) {
-	// noop
+	fs.StringVar(&f.Config, "config", "", "Path to a file containing a WebhookConfiguration object used to configure the webhook")
```

Comment:
Yes, docs will be how users work out the config file format (and examples). In future if/when we support loading this config from a CRD, this will change and be discoverable via `kubectl explain` :) (this is the same as other k8s components today, for what it is worth)